cl/visualizations/static/js/scotus_map.js

```diff
@@ -1237,8 +1237,8 @@ $(document).ready(function () {
 		var value = 'no data';
 
 		if (s !== '') {
-			value = '<a href="http://supremecourtdatabase.org/analysisCaseDetail.php?cid=' +
-				s + '" target="_blank">' + s + '</a>' +
+			value = '<a href="http://scdb.wustl.edu/analysisCaseDetail.php?cid=' +
+				s + '"-01 target="_blank">' + s + '</a>' +
```

Comment:
@Colinstarger Looks like this was wrong, but luckily it's not causing any damage. The link this forms is:

    <a href="http://scdb.wustl.edu/analysisCaseDetail.php?cid=1918-168" -01="" target="_blank">1918-168</a>

So, I tweaked this to get a quick fix. The correct code should be:

    s + '-01" target="_blank">' + s + '</a>' +